sources/packages/web/src/components/generic/TooltipIcon.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-tooltip :location="location">
+    <template #activator="{ props }">
+      <v-icon
+        icon="fa:fas fa-circle-question"
+        class="tooltip-icon mx-2"
+        v-bind="props"
+      />
+    </template>
+    <slot></slot>
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+export default {
+  props: {
+    activator: {
```

Comment:
mm, this prop is not used in the template

Reply:
That property is defaulted with parent and kept for the purpose of overriding if required.

Reply:
I think if it's for the future. better to remove it now. and when the required come, we can adjust